configs/warp_routes/USDT/ethereum-viction.yaml

```diff
@@ -0,0 +1,29 @@
+viction:
+  mailbox: "0x2f2aFaE1139Ce54feFC03593FeE8AB2aDF4a85A7"
+  owner: "0x23ed65DE22ac29Ec1C16E75EddB0cE3A187357b4"
+  ownerOverrides:
+    proxyAdmin: "0x23ed65DE22ac29Ec1C16E75EddB0cE3A187357b4"
+    validatorAnnounce: "0xa7ECcdb9Be08178f896c26b7BbD8C3D4E844d9Ba"
+    testRecipient: "0xa7ECcdb9Be08178f896c26b7BbD8C3D4E844d9Ba"
+    fallbackRoutingHook: "0xa7ECcdb9Be08178f896c26b7BbD8C3D4E844d9Ba"
+    _icaAddress: "0x23ed65DE22ac29Ec1C16E75EddB0cE3A187357b4"
+  type: synthetic
+  name: USDT
+  symbol: USDT
+  decimals: 6
+  totalSupply: 0
+  gas: 75000
+  interchainSecurityModule: "0x0000000000000000000000000000000000000000"
```

Comment:
Can we exclude the interchainSecurityModule being 0x0 part? Happy for that to be in a follow up PR

Reply:
Will follow up https://github.com/orgs/hyperlane-xyz/projects/1/views/8?pane=issue&itemId=94647709